guides/security/acceptable_use_policy.md

```diff
@@ -0,0 +1,76 @@
+
+Below are the minimum expectations for accessing those services appropriate to your role and responsibilities.
+
+## I will act with integrity at all times
+
+I will use the Made Tech service for the purpose it is intended for at all times, and not do anything knowingly that could harm the integrity or reputation of Made Tech.
+
+## I will use any Made Tech issued devices properly
+I understand Made Tech issued devices are issued for work purposes and I agree to return them for inspection as and when required. I will return it on leaving the employment of Made Tech or when it is no longer required unless otherwise agreed with Made Tech prior to leaving.
+
+## Personal Use
+I will exercise good judgement at all times regarding the reasonableness of any personal use
+
+## I will enable  2-factor authentication where this is required
+2-factor authentication (2FA) requires two forms of authentication, often a password and a code, sometimes sent via SMS or an application. It helps protect user accounts in the case that your password is somehow exposed. 
+
+## I will ensure my laptop isn’t left unattended or unlocked
+Lock your screen or put your computer to sleep when you’re not around. Colleagues might not have the same permissions as you and visitors might be wandering around where you’re working. When not in use ‘at work’, you can store in the laptop lockers provided or ‘at home’, in a safe place where it can’t be seen. If you are using your device in a public place, don’t leave it unattended and be wary of those who may be watching over your shoulder. 
+
+## I will ensure I report the arrival of guests and visitors promptly
+We keep a record of all visitors to the Made Tech office. All visitors should be logged using the appropriate system and you should alert the member of the team they are here to visit. If someone tries to enter the office and you feel uncomfortable, you are within your rights to challenge their reasons for entry (gently).
+
+## If my device is stolen or missing, I immediately report it
+Lots of information can be extracted from stolen or misplaced laptops, even in some cases when the disk has been encrypted. Report a missing machine quickly, better safe than sorry. We use remote device management to remotely wipe data from your device in the event that it is stolen. You should report all stolen items to the IMS Manager or CSO, who will record the incident. 
+
+## I will backup my work in the right place
+Information saved on a device is at risk of loss or theft and will not be backed up. To protect against theft, laptops are encrypted. Made Tech provides back up facilities within Google (Drive). Each team at Made Tech has their own Team Drive and it is the responsibility of those teams to maintain what is kept in those drives. All data must be copied to these locations to ensure there is no data loss in the event of your laptop becoming unusable.
```

Comment:
`Google (Drive)` is different to how we refer to G-Suite elsewhere
We could say `Made Tech backs up data stored on G-Suite, GitHub, and Slack.`

Reply:
amended